### PR TITLE
PREL-101 Update Installation doc

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/percona-release.rst

### CODE REVIEW

File: source/percona-release.rst [hunk 1]
```diff
@@ -45,12 +45,12 @@ official package:
       $ sudo apt-get update
       $ sudo apt-get install -y wget gnupg2 lsb-release
 
-1. Fetch the repository package:
+1. Fetch the repository package. You need to run this script as root or with :program:`sudo`. :
```
Comment (EvgeniyPatlan) [inline source/percona-release.rst:48]: why? it works well without root permissions

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: source/percona-release.rst (hunk 1 above)
Comment (patrickbirch) [reply source/percona-release.rst:48]: Fixed.

Comment (patrickbirch) [reply source/percona-release.rst]: On the Ubuntu 20.04 (Docker) version I used to test, it was also missing CURL. Should I add that?

File: source/percona-release.rst [hunk 2]
```diff
@@ -42,20 +42,20 @@ official package:
 
    .. code-block:: bash
 
-      $ sudo apt-get update
-      $ sudo apt-get install -y wget gnupg2 lsb-release
+      $ apt-get update
+      $ apt-get install -y wget gnupg2 lsb-release
 
-1. Fetch the repository package:
+1. Fetch the repository package.
```
Comment (EvgeniyPatlan) [inline source/percona-release.rst:48]:
```suggestion
1. Fetch the repository package:
```

File: source/percona-release.rst [hunk 3]
```diff
@@ -42,20 +42,20 @@ official package:
 
    .. code-block:: bash
 
-      $ sudo apt-get update
-      $ sudo apt-get install -y wget gnupg2 lsb-release
+      $ apt-get update
+      $ apt-get install -y wget gnupg2 lsb-release
```
Comment (EvgeniyPatlan) [inline source/percona-release.rst:46]:
```suggestion
      $ apt-get install -y wget gnupg2 lsb-release curl
```
As I remember @nastena1606 already added it previously but better to recheck

File: source/percona-release.rst [hunk 4]
```diff
@@ -42,20 +42,20 @@ official package:
...
              
       $ wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
-
+      
 #. Install the downloaded repository package with ``dpkg``:
 
    .. code-block:: bash
 		   
-      $ sudo dpkg -i percona-release_latest.generic_all.deb
+      $ dpkg -i percona-release_latest.generic_all.deb
```
Comment (EvgeniyPatlan) [inline source/percona-release.rst:58]: sudo is needed for package installation

File: source/percona-release.rst [hunk 5]
```diff
@@ -187,50 +192,48 @@ Available commands are ``enable``, ``enable-only``, ``disable``, ``setup`` and `
...
+     - ``ppg-12.3``
+     - ``pdmdb-4.2``
+     - ``pdmdb-4.2.6``
+     - ``pdmdb-4.2.7``
+     - ``pdmdb-4.2.8``
+     - ``pdps-8.0.19``
+     - ``pdpxc-8.0.19``
+     - ``pdps-8.0.20``
+     - ``pdps-8.0``
+     - ``pdpxc-8.0``
```
Comment (EvgeniyPatlan) [inline source/percona-release.rst:224]: it is better to make rebase as the list isn't full

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: source/percona-release.rst (hunk 2 above)
Comment (patrickbirch) [reply source/percona-release.rst:48]: Fixed.

File: source/percona-release.rst (hunk 3 above)
Comment (patrickbirch) [reply source/percona-release.rst:46]: Fixed.

File: source/percona-release.rst (hunk 4 above)
Comment (patrickbirch) [reply source/percona-release.rst:58]: OK. Added back.

**Pull request closed without merging**